Create ui-c8y-1022-19-0-send-reset-password-link-from-user-details-view.md (#3640)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1022-19-0-send-reset-password-link-from-user-details-view.md

---
date: ""

> title: Send reset password link from user details view

Reviewer: ```suggestion
title: Send reset password link from user details
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63951
version: 1022.19.0
---

> This new feature allows User Admin to sent reset password links from existing user details view.

Reviewer: ```suggestion
To simplify the user management process, it is now possible for the Admin user of a tenant to send a reset password link from the existing user details view.
```

> To simplify the user management process, it is now possible for the Admin user of a tenant to send a reset password link from the existing user details view.

Reviewer: ```suggestion
To simplify the user management process, it is now possible for the Admin user of a tenant to send a reset password link from the user details of an existing user.
```
